CashBudTests/Mocks/Services/LocalPersistenceServiceMock.swift

```diff
@@ -6,3 +6,28 @@
 //
 
 import Foundation
+
+class LocalPersistenceServiceMock: LocalPersistenceServiceProtocol {
+    // TODO: Implement mock methods for local persistence service
+    
```

Comment:
add all functions according to the LocalPersistenceServiceProtocol and implement them

Reply:
add all functions from the LocalPersistenceServiceProtocol and implement them

Reply:
add all functions according to the LocalPersistenceServiceProtocol and implement them